compiler/interface.go

```diff
@@ -84,6 +84,27 @@ func (b *builder) extractValueFromInterface(itf llvm.Value, llvmType llvm.Type)
 	return b.emitPointerUnpack(valuePtr, []llvm.Type{llvmType})[0]
 }
 
+func (c *compilerContext) pkgPathPtr(pkgpath string) llvm.Value {
+	pkgpathName := "reflect/types.type.pkgpath.empty"
+	if pkgpath != "" {
+		pkgpathName = "reflect/types.type.pkgpath:" + pkgpath
+	}
+
+	pkgpathInitializer := c.ctx.ConstString(pkgpath+"\x00", false)
```

Comment:
Note that `c.ctx.ConstString(pkgpath, true)` would do the same.
(I don't really care about which it is, the latter might be more readable).